pipelines/infrastructure/cloud_custodian_policies/sync_ec2_tags.yaml

```diff
@@ -0,0 +1,52 @@
+---
+policies:
+- name: find-mitodl-ec2-instances
+  resource: aws.ec2
+  description: |
+    Find all the running ec2 instances that were spawned by an asg, belong of mitodl, and do not belong to ike. Take no action. Output used for next filter.
+  filters:
+  - State.Name: running
+  - "tag:OU": present
+  - "tag:aws:autoscaling:groupName": present
+  - not:
+    - "tag:OU": "ike"
+
+- name: find-and-propagate-tags-mitodl-ebs-volumes
+  resource: aws.ebs
+  description: |
+    Finds and propagates tags from ec2 instances found in the previous policy and adds those tags to the block devices on each instance.
+  filters:
+  - type: value
+    key: "VolumeId"
+    op: in
+    value_from:
+      # TODO, will need to figure out what the concourse path will be here?
+      # Include a comment on how to run it locally?
```

Comment:
Should we make these separate steps in the pipeline so that we can use the input/output? Or should this be a relative path in the assumption that the previous step will make the file in the local directory?